### PR TITLE
docs/contributing-readme-update

### PR DESCRIPTION
**Pull request by KacperMadej**
Updated Contributing.md and some readme files.

### CODE REVIEW

**Review by MarkusBarstad — APPROVED** [review]
Well done!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,48 +2,79 @@
...
 
+### Test a fix
+
+When an issue is resolved, we commit a fix and mark the issue closed. This
+doesn't mean that a new release is available with the fix applied, but that it
+is fixed in the development code and will be added to the next stable release.
+Stable versions are typically released every 1-3 months. To try out the fix
+immediately, you can run http://github.highcharts.com/highcharts.js or
+http://github.highcharts.com/highstock.js from any website, but do not use these
+URLs in production.
```
Comment (goransle) [inline CONTRIBUTING.md:55]: Perhaps we should advertise the nightly here instead/as well?

[jsfiddle for convenience](https://jsfiddle.net/goransle/58nutgqz/)

**Review by goransle — APPROVED** [review]
Nice, just a thought re preview code:

**Review by TorsteinHonsi — APPROVED** [review]
Thanks!

**Pull request merged**